concent_api/utils/fields.py

```diff
@@ -0,0 +1,24 @@
+import base64
+
+from django.db import models
+
+
+class Base64Field(models.TextField):
+    """ Base64 encoding field for storing binary data in Django TextFields. """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.field_name = None
+
+    def contribute_to_class(self, cls, name, private_only=False, virtual_only=models.fields.NOT_PROVIDED):
+        if self.db_column is None:
+            self.db_column = name
+        self.field_name = name + '_base64'
+        super().contribute_to_class(cls, self.field_name, private_only=private_only, virtual_only=virtual_only)
+        setattr(cls, name, property(self.get_data, self.set_data))
+
+    def get_data(self, obj):
+        return base64.b64decode(getattr(obj, self.field_name))
+
+    def set_data(self, obj, data):
+        setattr(obj, self.field_name, base64.b64encode(data))
```

Comment:
Is this decoding/encoding really necessary here? In the views you assign values that are already base64-encoded (because the client sends them in an encoded form) so aren't you encoding them twice?

Reply:
Yes, without it it is throwing:
`django.utils.encoding.DjangoUnicodeDecodeError: 'utf-8' codec can't decode byte 0xed in position 1:`